core/client/src/test/java/alluxio/client/file/FileOutStreamTest.java

```diff
@@ -246,9 +258,15 @@ public void cancel() throws Exception {
       Assert.assertTrue(mAlluxioOutStreamMap.get(streamIndex).isCanceled());
     }
     // Don't persist or complete the file if the stream was canceled
-    verify(mFileSystemMasterClient, times(0)).completeFile(FILE_NAME,
-        CompleteFileOptions.defaults());
-    verify(mUnderFileSystem).delete(anyString(), eq(false));
+    if (mDelegationEnabled) {
```

Comment:
could we break this into two tests, one with delegation enabled and the other with delegation disabled?